client/src/parser.js

```diff
@@ -0,0 +1,99 @@
+const { parse } = require('path');
+
+/**
```

Comment:
There are two functions in this file, but only one documentation for a function. Please put the documentation directly in front of the function which it is supposed to describe and document both functions